doc/Team.txt

```diff
@@ -3,5 +3,4 @@ dtran1 darrentran
 tan4 toado
 jtha jameshahaha
 tbojovic bojotamara
-tntran1 Ny-Tran
-
+tntran1 Ny-Tran
```

Comment:
anyway to get rid of this lol?

Reply:
He'll just have to get rid of the extra line at the end of the document

Reply:
added the extra line again LOL